components/bl-calendarHeatmap-component/src/helpers.js

```diff
@@ -0,0 +1,52 @@
+export const validate = (items) => {
+  if (typeof items === 'string') {
+    return items.split(',');
+  }
+
+  return items;
+};
+
+export const shiftDate = (date, numDays) => {
+  const newDate = new Date(date);
+
+  newDate.setDate(newDate.getDate() + numDays);
+
+  return newDate;
+};
+
+export const shadeColor = (col, amt) => {
+  let usePound = false;
+
+  if (col[0] === '#') {
+    col = col.slice(1);
+    usePound = true;
+  }
+
+  const num = parseInt(col, 16);
+
+  let r = (num >> 16) + amt;
+
+  if (r > 255) {
+    r = 255;
+  } else if (r < 0) {
+    r = 0;
+  }
+
+  let b = ((num >> 8) & 0x00FF) + amt;
```

Comment:
@Valodya, please accept it.

Reply:
I accept it =)
but, please add a comment with the link where the code comes from